docs/testing/integration_tests.rst

```diff
@@ -2,4 +2,50 @@
 Integration Tests
 *****************
 
-- To be completed
+
+Configuration
+=============
+
+The main configuration file to look at is ``/tests/protractor.conf.js``.
+It configures our **specs**, **exclude** and **suites** paths.
+
+Inside **specs** we specify the location of all specs that should be launched.
+It can be set to run all tests that are in the specs folder like this ``specs: ['specs/*.js']``.
+
+The **exclude** setting allows us to specifically exclude certain specs from
+running. By defining a path - say ``specs/spec.log*.js`` in ``exclude`` - we
+can exclude scripts such as ``specs/spec.login.js`` and ``specs/spec.logout.js``
+from being launched.
+
+Alternatively, **suites** may be used - we can group spec files into suites
+with the keyword. For example we can add 2 tests into suite like this:
+
+.. code-block:: javascript
+
+    // Spec patterns are relative to the location of the spec file. They may
+    // include glob patterns.
+    suites: {
```

Comment:
so what should be the file organisation?

```
/tests/integration/
  - spec.article.create.js
  - spec.article.delete.js
  - spec.login.js
  - spec.logout.js
```

and than the rest is managed through configuration?